meetings/2017-06-08.md

```diff
@@ -0,0 +1,83 @@
+# Node.js Foundation Community Committee Meeting 2017-06-08
+
+## Links
+
+* **GitHub Issue**: [community-committee#62](https://github.com/nodejs/community-committee/issues/62)
```

Comment:
Would it be possible to start including the video recording link in the Links section here?